### ISSUE TITLE
Enable duplicate labels on UI

### ISSUE THREAD

**Issue by Michaeasmith** [robotmoose/robotmoose#59]
Currently, if you try to add a label to the UI with the same string as an existing label, it will do nothing and show no error message.

This is a problem in some cases. For example, if you are using labels to display multiple different numbers (e.g. sensor readings), one will not display if it is the same number as another label.

Buttons and checkboxes behave the same way. Should they be changed as well?

**Comment by Arsh25** [comment]
I talked to @mrmoss about this way back and apparently this was a conciois decision to not allow multiple UI elements with the same name.

**Comment by mrmoss** [comment]
Yes, this is the same bug that doesn't update the UI buttons and such. The
UI stores a list of UI elements, if the element's value already exists,
then it isn't created.

We should discuss this in detail at the next meeting (Wednesday) and decide
exactly how this should function.
On Jun 14, 2016 22:31, "Arsh Chauhan" notifications@github.com wrote:

@mikemoss https://github.com/mikemoss

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
https://github.com/robotmoose/robotmoose/issues/59#issuecomment-226099816,
or mute the thread
https://github.com/notifications/unsubscribe/ABvm0zYrXfdrJEg2Brx8Sttcqvba0-rtks5qL5xLgaJpZM4I2CKE
.

**Comment by Arsh25** [comment]
Closed duplicate of #58

**Comment by FineTralfazz** [comment]
Should this be closed, or did we decide that we wanted to change how this works?

**Comment by mrmoss** [comment]
We defintely need duplicate labels...and I think we settled on a clear
button as well? As far as I know this is still a problem...
